app/models.py

```diff
@@ -0,0 +1,80 @@
+from . import db
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer,
+                          SignatureExpired, BadSignature)
+from flask import current_app
+
+
+class User(db.Model):
+    __tablename__ = 'users'
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(255), unique=True, nullable=False)
+    email = db.Column(db.String(255), nullable=False)
+    pass_hash = db.Column(db.String(255), nullable=False)
+    bucketlist = db.relationship('Bucketlist', backref='user', lazy='dynamic',
+                                 cascade='all, delete-orphan')
+
+    @property
+    def password(self):
+        raise AttributeError('password is not a readable attribute')
+
+    @password.setter
+    def password(self, password):
+        self.pass_hash = generate_password_hash(password)
+
+    def verify_password(self, password):
+        return check_password_hash(self.pass_hash, password)
+
+    def generate_confirmation_token(self, expiration=3600):
+        s = Serializer(current_app.config['SECRET_KEY'], expiration)
+        return s.dumps({'id': self.id})
+
+    @staticmethod
+    def token_confirm(token):
+        s = Serializer(current_app.config['SECRET_KEY'])
+        try:
+            data = s.loads(token)
+        except SignatureExpired:
+            """ valid token but expired """
+            return False
+        except BadSignature:
+            """ invalid token """
+            return False
+        user_id = data['id']
+        return user_id
+
+    def __repr__(self):
+        return "<User %s>" % self.username
+
+
+class Bucketlist(db.Model):
+    __tablename__ = 'bucketlists'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    date_created = db.Column(db.DateTime, nullable=False,
+                             default=datetime.utcnow)
+    date_modified = db.Column(db.DateTime, default=datetime.utcnow,
+                              onupdate=datetime.utcnow)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+    items = db.relationship('Item', backref='bucketlist', lazy='dynamic',
+                            cascade='all, delete-orphan')
+
+    def __repr__(self):
+        return '<Bucketlist %s>' % self.name
+
+
+class Item(db.Model):
+    __tablename__ = 'items'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    date_created = db.Column(db.DateTime, nullable=False,
+                             default=datetime.utcnow)
+    date_modified = db.Column(db.DateTime, default=datetime.utcnow,
+                              onupdate=datetime.utcnow)
+    status = db.Column(db.Boolean, default=False)
+    bucket_id = db.Column(db.Integer, db.ForeignKey('bucketlists.id'),
+                          nullable=False)
+
```

Comment:
In your three models you have common fields. To avoid repeating fields like `id`, `name`, `date-created` e.t.c. You can create a class called base model that is inherited by all the other models.
Checkout what i did here.. https://github.com/andela-dmigwi/A-BucketList/blob/master/app/models.py if you need in understanding why the code  am available.